requirements.txt

```
@@ -0,0 +1,23 @@
+openvino-dev[onnx,tensorflow2]==2021.4
```

```suggestion
openvino-dev[onnx,tensorflow2]~=2021.4
```
Since 2021.4 is an LTS release, and since we plan to tag a notebook release, we should use `~=` so security patches/bug fixes in 2021.4.X will be available to someone who uses this requirements file in the future.

---

Good point. `~=` pins major versions though, so `~=2021.4` is also satisfied by 2021.3. I'll change it to `==2021.4.*`